apps/web/components/twitter-editor.tsx

```diff
@@ -258,14 +258,14 @@ export default function TwitterEditor() {
         </div>
         <div className="flex flex-col mt-4 border p-2 rounded-md">
           <label className="block mb-2">
-            Preview Height: {previewHeight ? previewHeight : 0}px
+            Preview Height: {Height ? Height : 0}px
           </label>
           <input
             type="range"
             min="40"
             name="previewHeight"
```

Comment:
name has to change as well as per the variable name because it wont update value otherwise